### PR TITLE
Fixed up getscinfo for ceased sidechains

### PR DESCRIPTION
**Pull request by abi87**
Fixed up scInfo for ceased sidechain, in order to keep epoch and end epoch height locked to ceasing epoch for ceased sidechains.
It would close issues https://github.com/HorizenOfficial/zend_oo/issues/38 https://github.com/HorizenOfficial/zend_oo/issues/42 and https://github.com/HorizenOfficial/zend_oo/issues/44

### CODE REVIEW

**Review by alsala — APPROVED** [review]

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -1032,128 +1032,250 @@ UniValue reconsiderblock(const UniValue& params, bool fHelp)
...
     if (!scView.GetSidechain(scId, scInfo)) {
         LogPrint("sc", "scid[%s] not yet created\n", scId.ToString() );
-        return false;
+        throw JSONRPCError(RPC_INVALID_PARAMETER, string("scid not yet created: ") + scId.ToString());
     }
     CSidechain::State scState = scView.isCeasedAtHeight(scId, chainActive.Height() + 1);
 
-    AddScInfoToJSON(scId, scInfo, scState, sc);
-    return true;
+    return FillScRecordFromInfo(scId, scInfo, scState, scRecord, bOnlyAliv, bVerbosee);
```
Comment (abi87) [inline src/rpc/blockchain.cpp:1097]: two typos here `bOnlyAliv` which should be `bOnlyAlive` and `bVerbosee` which should be `bVerbose`

File: src/rpc/blockchain.cpp [hunk 2]
```diff
@@ -1032,128 +1032,250 @@ UniValue reconsiderblock(const UniValue& params, bool fHelp)
...
+    else
+    {
+        int from = 0;
+        if (params.size() > 3)
+    	    from = params[3].get_int();
 
-    // dump all of them if any
-    UniValue result(UniValue::VARR);
-    AddScInfoToJSON(result);
+        int to = 50;
```
Comment (abi87) [inline src/rpc/blockchain.cpp]: as discussed AFK, `to` default should be `-1`

File: src/rpc/blockchain.cpp (hunk 1 above)
Comment (alsala) [reply src/rpc/blockchain.cpp]: Done

File: src/rpc/blockchain.cpp (hunk 2 above)
Comment (alsala) [reply src/rpc/blockchain.cpp]: Done

**Comment by abi87** [comment]
Absorbed by https://github.com/HorizenOfficial/zend_oo/pull/74

**Pull request merged**